tests/components/webostv/test_media_player.py

```diff
@@ -483,6 +469,8 @@ async def test_client_disconnected(
     async_fire_time_changed(hass, dt_util.utcnow() + timedelta(seconds=20))
     await hass.async_block_till_done()
 
+    assert "TimeoutError" not in caplog.text
```

Comment:
I have verified that this test fails if the errors are not suppressed at https://github.com/home-assistant/core/blob/6642a71f2951148fd50af5aaab9b55037d673d50/homeassistant/components/webostv/media_player.py#L328